uint8 dataID
uint8 mode
uint8 teached_pos
uint8 active_axis
int16 axis_velocity
int16 target_x
int16 target_y
int16 target_z
int16 target_roll
int16 target_pitch
int16 target_yaw
uint8 gripper_status
uint8 movementStarted
uint8 collision
uint8 dummy0
uint8 dummy1
uint8 dummy2